pkgs/development/compilers/cudatoolkit/redist/overrides.nix

```diff
@@ -38,13 +61,13 @@ in (lib.filterAttrs (attr: _: (prev ? "${attr}")) {
 
   nsight_compute = prev.nsight_compute.overrideAttrs (oldAttrs: {
     nativeBuildInputs = oldAttrs.nativeBuildInputs
-    ++ (if (lib.versionOlder prev.nsight_compute.version "2022.2.0")
-       then [ pkgs.qt5.wrapQtAppsHook ]
-       else [ pkgs.qt6.wrapQtAppsHook ]);
+      ++ (if (lib.versionOlder prev.nsight_compute.version "2022.2.0")
+    then [ pkgs.qt5.wrapQtAppsHook ]
+    else [ pkgs.qt6.wrapQtAppsHook ]);
     buildInputs = oldAttrs.buildInputs
-    ++ (if (lib.versionOlder prev.nsight_compute.version "2022.2.0")
-       then [ pkgs.qt5.qtwebview ]
-       else [ pkgs.qt6.qtwebview ]);
+      ++ (if (lib.versionOlder prev.nsight_compute.version "2022.2.0")
+    then [ pkgs.qt5.qtwebview ]
+    else [ pkgs.qt6.qtwebview ]);
```

Comment:
purely formatting?

Reply:
@samuela Oh, that's from `nixpkgs-fmt`. Want me to revert?

Reply:
yeah best to preserve the git blame I think

Reply:
Reverted and rebased ✅ 
But for for git-blames not to be an issue in the future, I think it would be a nice convention to always run `nixpkgs-fmt` before committing